bugbear.py

```diff
@@ -1104,6 +1105,29 @@ def check_for_b022(self, node):
         ):
             self.errors.append(B022(node.lineno, node.col_offset))
 
+    def check_for_b908(self, node: ast.With):
+        has_multiline_body = len(node.body) > 1
+        for node_item in node.items:
+            if (
+                not hasattr(node_item.context_expr, "func")
+                or not hasattr(node_item.context_expr.func, "value")
+                or not hasattr(node_item.context_expr.func.value, "id")
+            ):
+                continue
+            if (
```

Comment:
I think we should also catch plain `with raises():`: https://grep.app/search?q=from%20pytest%20import%20raises

Also, let's cover `pytest.warns` too, it's the same idea.